static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -425,14 +426,22 @@ export function ServerSideSampling({project}: Props) {
           )}
         />
         {!!rules.length && !fetchingRecommendedSdkUpgrades && (
-          <SamplingSDKAlert
+          <SamplingSDKUpgradesAlert
             organization={organization}
             projectId={project.id}
             rules={rules}
             recommendedSdkUpgrades={recommendedSdkUpgrades}
             onReadDocs={handleReadDocs}
           />
         )}
+        {!!rules.length && (
+          <SamplingSDKClientRateChangeAlert
```

Comment:
We probably don't want to show this alert if we show `SamplingSDKUpgradesAlert`.